pkg/sql/compile/compile.go

```diff
@@ -329,6 +331,8 @@ func (c *Compile) Run(_ uint64) error {
 		c.proc.TxnOperator.ResetRetry(false)
 	}
 	if err := c.runOnce(); err != nil {
+		c.fatalLog(0, err)
```

Comment:
What is this for?

Reply:
panic if meetIf, in pessimistic RC mode, an error is encountered that is not expected, panic